pycds/alembic/versions/3d50ec832e47_add_constraint_on_meta_vars.py

```diff
@@ -0,0 +1,57 @@
+"""add constraint on meta vars
+
+Revision ID: 3d50ec832e47
+Revises: 0d99ba90c229
+Create Date: 2022-01-12 13:11:12.480424
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import citext as ci
+from pycds.orm.views.version_84b7fc2596d5 import ObsWithFlags
+
+# revision identifiers, used by Alembic.
+revision = "3d50ec832e47"
+down_revision = "0d99ba90c229"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ### #
+   op.execute("CREATE EXTENSION IF NOT EXISTS citext")
+   
+   op.drop_replaceable_object(ObsWithFlags)
+   op.alter_column(
+        "meta_vars",
+        "net_var_name",
+        type_=ci.CIText(),
+        schema="crmp",
```

Comment:
You must not specify a fixed schema name here. Look in other migrations for how to code it to use an externally specified schema (specified by env var `PYCDS_SCHEMA_NAME`).

Reply:
FYI, the tests run with `PYCDS_SCHEMA_NAME=other` just to make sure we don't miss this.